python/pyspark/sql/streaming/stateful_processor.py

```diff
@@ -69,6 +69,60 @@ def clear(self) -> None:
         self._value_state_client.clear(self._state_name)
 
 
+class TimerValues:
+    """
+    Class used for arbitrary stateful operations with transformWithState to access processing
+    time or event time for current batch.
+
+    .. versionadded:: 4.0.0
+    """
+    def __init__(
+            self,
```

Comment:
Shall we leave Github to resolve the review comment rather than manually marking as resolved? I don't see any new commit to resolve these style comments. I guess you've addressed but missed to push commit, but easier to track if we resolve the comment as "outdated".

Reply:
(I tend to use reaction to distinguish comments I agree to address, especially style comments.)